pkg/daemon/ceph/client/mgr.go

```diff
@@ -6,9 +6,18 @@ import (
 	"github.com/rook/rook/pkg/clusterd"
 )
 
-// MgrEnableModule Enable module for mgr
+// MgrEnableModule enables a mgr module
 func MgrEnableModule(context *clusterd.Context, clusterName, name string, force bool) error {
-	args := []string{"mgr", "module", "enable", name}
+	return enableModule(context, clusterName, name, force, "enable")
+}
+
+// MgrDisableModule disables a mgr module
+func MgrDisableModule(context *clusterd.Context, clusterName, name string) error {
+	return enableModule(context, clusterName, name, false, "disable")
+}
+
+func enableModule(context *clusterd.Context, clusterName, name string, force bool, action string) error {
```

Comment:
this function name is a bit misleading since it can be used for disabling as well, but the function name and none of the parameters are named to indicate that. not a huge deal though.

Reply:
yeah, it did occur to me to rename the function after its purpose changed. I'll get it next time i'm in there.